code/src/OSFramework/Configuration/Column/ColumnConfig.ts

```diff
@@ -25,7 +27,6 @@ namespace OSFramework.Configuration.Column {
         public header: string;
         public isMandatory: boolean;
         public multiLine: boolean;
-        public required: boolean;
```

Comment:
Be careful when you remove this property, if I remember correctly, it has a direct impact on the deletion of the cell's content. So if you have the required = true, you won't be able to delete the content of any cell from that column. If you set it to false, you will delete the content of the cells from that column just by pressing DEL key.